### ISSUE TITLE
Сообщения: апгрейд 04.22

### ISSUE THREAD

**Issue by nicothin** [nicothin/nesia#18]

**Comment by nicothin** [comment]
@vladimir075 Есть вопросы:

1. раздел "Вопрос от продавца": в области переписки есть текст "You can contact free with James if user will buy acsess or you can unlock contacts". На большой ширине есть, на малой нет. Это так и задумано?
2. раздел "Вопрос от продавца": на большой ширине в шапке есть кнопка Unlock contacts, на малой ширине нет. Так и задумано?

**Comment by vladimir075** [comment]
@nicothin Николай, а есть ссылочка сверстанного по обычаю? )

**Comment by vladimir075** [comment]
@nicothin Николай, а еще вопросик хоть и немного не по теме в данном thread
@nadezdafrolova Надежда дизайнер спрашивает у какая у нас сетка в Терре. Помню, что у нас с этим немного путаница, поскольку изначально верстали Терру другие, но тем не менее, у Вас наверное больше инфы по этому вопросу. Сможете ответить тут Наде?

**Comment by nicothin** [comment]
@vladimir075 

> https://nicothin.pro/nesia/messages.html

конечно: https://nicothin.pro/nesia/messages.html (нужно ткнуть в пункт "Вопрос от продавца")


> @nadezdafrolova Надежда дизайнер спрашивает у какая у нас сетка в Терре.

Там bootstrap 3.0, сетка 12- колоночная резина (https://getbootstrap.com/docs/3.4/css/#grid - в блоке любой ширины можно организовать внутреннюю 12-колочную сетку, а вот сделать одинаковую ширину ячеек внутри разных по ширине родителей, когда разница не кратна 1/12 ширины одного из них, нельзя).

@greeceruit да, видимо, оптимально будет при к-л проблемах прицельно взаимодействовать именно по ним.
Используемая мной методология вёрстки, тоже, подразумевает компонентный подход, конечно.
Единственная возможная проблема: при выпиливании с проекта к-л разметки нужно убирать и её стили. иначе CSS-файл будет постепенно расти. Есть, кажется, инструменты автоматизации такой оптимизации, но с закрытыми частыми сайтов применить их сложновато.

**Comment by vladimir075** [comment]
@nicothin Николай, прошу сделать здесь небольшую правку. 
1) В мобильной версии сверху чата, чтобы появлялось красным гиперлинк к какому объекту относится чат (в том же месте что он появляется и десктопе) 
![Screen Shot 2022-09-03 at 18 34 31](https://user-images.githubusercontent.com/17883276/188277800-1aa0dee5-ef21-4816-8b92-416a221bbee1.png)
2) Заголовок ответа в середине чата (там где серым шрифтом) должен содержать в себе также гиперлинк красным шрифтом (это для мобилы) - это есть в дизайне.
Кстати, по последней задаче, там где у нас в это заголовок добавился еще колокольчик, это вариация (не все сообщения с этим колокольчиком), так что надо два варианта верстки этого экрана, с колокольчиком и без.

**Comment by nicothin** [comment]
по пункту 1 - https://github.com/nicothin/nesia/issues/35

**Comment by nicothin** [comment]
по пункту 2 - добавил ссылку красного цвета для демки

https://nicothin.pro/nesia/messages.html, подраздельчик "Положительный ответ..." с меткой "2" (третий сверху)

**Comment by nicothin** [comment]
по поводу колокольчика: иконка добавлена прямо в разметке, если ее там не будет - ok, будет текст без иконки (по-прежнему выровненный по центру)
